<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="mainFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>780</width>
      <height>360</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QListWidget" name="listaObjetos">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>0</y>
      <width>171</width>
      <height>201</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>171</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>171</width>
      <height>16777215</height>
     </size>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>581</width>
      <height>134</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_15">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Window</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="font">
                  <font>
                   <pointsize>10</pointsize>
                  </font>
                 </property>
                 <property name="layoutDirection">
                  <enum>Qt::LeftToRight</enum>
                 </property>
                 <property name="text">
                  <string>X</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                 <property name="buddy">
                  <cstring>spinWindowX</cstring>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="spinWindowX"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="font">
                  <font>
                   <pointsize>10</pointsize>
                  </font>
                 </property>
                 <property name="layoutDirection">
                  <enum>Qt::LeftToRight</enum>
                 </property>
                 <property name="text">
                  <string>Y</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                 <property name="buddy">
                  <cstring>spinWindowY</cstring>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="spinWindowY"/>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Rotacionar</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="spinWindowRotacao"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Escalonar</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="spinWindowEscala"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Largura</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="spinWindowLargura"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="label_7">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Altura</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="spinWindowAltura"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="lineWidth">
        <number>3</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Objeto</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_9">
               <item>
                <widget class="QLabel" name="label_10">
                 <property name="font">
                  <font>
                   <pointsize>10</pointsize>
                  </font>
                 </property>
                 <property name="layoutDirection">
                  <enum>Qt::LeftToRight</enum>
                 </property>
                 <property name="text">
                  <string>X</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                 <property name="buddy">
                  <cstring>spinWindowX</cstring>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="spinObjX">
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_10">
               <item>
                <widget class="QLabel" name="label_11">
                 <property name="font">
                  <font>
                   <pointsize>10</pointsize>
                  </font>
                 </property>
                 <property name="layoutDirection">
                  <enum>Qt::LeftToRight</enum>
                 </property>
                 <property name="text">
                  <string>Y</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                 <property name="buddy">
                  <cstring>spinWindowY</cstring>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="spinObjY">
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_11">
             <item>
              <widget class="QLabel" name="label_12">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Rotacionar</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="spinObjRotacao"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_12">
             <item>
              <widget class="QLabel" name="label_13">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Escalonar</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="spinObjEscala"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_13">
             <item>
              <widget class="QLabel" name="label_14">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Transladar X</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="spinObjTransX"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_14">
             <item>
              <widget class="QLabel" name="label_15">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Transladar Y</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="spinObjTransY"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>160</y>
      <width>271</width>
      <height>42</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_17">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_16">
       <item>
        <widget class="QPushButton" name="btnReta">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/imgs/imgs/reta.png</normaloff>:/imgs/imgs/reta.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnQuadrado">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/imgs/imgs/quadrado.png</normaloff>:/imgs/imgs/quadrado.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnTriangulo">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/imgs/imgs/triangulo.png</normaloff>:/imgs/imgs/triangulo.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="btnAplicar">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Aplicar</string>
       </property>
       <property name="shortcut">
        <string>Return</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>199</width>
      <height>20</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_18">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Largura da Caneta</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="tamanhoCaneta">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btnOcioso">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>160</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/imgs/imgs/clicker.png</normaloff>:/imgs/imgs/clicker.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>mainFrame</class>
   <extends>QFrame</extends>
   <header>mainframe.h</header>
   <container>1</container>
   <slots>
    <slot>addObjeto()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnReta</sender>
   <signal>released()</signal>
   <receiver>MainWindow</receiver>
   <slot>desenharLinha()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>353</x>
     <y>207</y>
    </hint>
    <hint type="destinationlabel">
     <x>298</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnQuadrado</sender>
   <signal>released()</signal>
   <receiver>MainWindow</receiver>
   <slot>desenharQuadrado()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>402</x>
     <y>189</y>
    </hint>
    <hint type="destinationlabel">
     <x>310</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnTriangulo</sender>
   <signal>released()</signal>
   <receiver>MainWindow</receiver>
   <slot>desenharTriangulo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>447</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel">
     <x>430</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnAplicar</sender>
   <signal>released()</signal>
   <receiver>MainWindow</receiver>
   <slot>aplicarTransformacoes()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>543</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>515</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listaObjetos</sender>
   <signal>currentItemChanged(QListWidgetItem*,QListWidgetItem*)</signal>
   <receiver>MainWindow</receiver>
   <slot>selecionarObj()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>645</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>607</x>
     <y>32</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnOcioso</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>pararDesenho()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>301</x>
     <y>214</y>
    </hint>
    <hint type="destinationlabel">
     <x>181</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>desenharLinha()</slot>
  <slot>desenharQuadrado()</slot>
  <slot>desenharTriangulo()</slot>
  <slot>aplicarTransformacoes()</slot>
  <slot>selecionarObj()</slot>
  <slot>pararDesenho()</slot>
 </slots>
</ui>
